rosdep/oe-opkg.yaml

```diff
@@ -0,0 +1,335 @@
+asio:
+  oe:
+    opkg:
+      packages: [asio@meta-oe]
+
+assimp:
+  oe:
+    opkg:
+      packages: [assimp@openembedded-core]
+
+boost:
+  oe:
+    opkg:
+      packages: [boost@openembedded-core]
+
+bzip2:
+  oe:
+    opkg:
+      packages: [bzip2@openembedded-core]
+
+cmake:
+  oe:
+    opkg:
+      packages: [cmake@openembedded-core]
+
+cppcheck:
+  oe:
+    opkg:
+      packages: [cppcheck-native@meta-sca]
+
+curl:
+  oe:
+    opkg:
+      packages: [curl@openembedded-core]
+
+eigen:
+  oe:
+    opkg:
+      packages: [libeigen@meta-oe]
+
+git:
+  oe:
+    opkg:
+      packages: [git@openembedded-core]
+
+gtk3:
+  oe:
+    opkg:
+      packages: [gtk+3@openembedded-core]
+
+libcairo2-dev:
+  oe:
+    opkg:
+      packages: [cairo@openembedded-core]
+
+libceres-dev:
+  oe:
+    opkg:
+      packages: [ceres-solver@meta-oe]
+
+libconsole-bridge-dev:
+  oe:
+    opkg:
+      packages: [console-bridge@meta-ros]
+
+libexpat1-dev:
+  oe:
+    opkg:
+      packages: [expat@openembedded-core]
+
+libfreetype6-dev:
+  oe:
+    opkg:
+      packages: [freetype@openembedded-core]
+
+libfreetype6:
```

Comment:
This is out of alphabetical order according to CI.

Reply:
Thanks; will adjust all in base.yaml and python.yaml .